librz/include/rz_rop.h

```diff
@@ -159,13 +161,14 @@ RZ_API void rz_core_rop_gadget_info_free(RZ_NULLABLE RzRopGadgetInfo *gadget_inf
 RZ_API void rz_core_rop_gadget_info_add_register(const RZ_NONNULL RZ_OUT RzRopGadgetInfo *gadget_info, RZ_NONNULL RzRopRegInfo *reg_info, bool is_dependency);
 RZ_API int rz_core_rop_gadget_info_update_register(RZ_INOUT RzRopGadgetInfo *gadget_info, RZ_NONNULL RzRopRegInfo *new_reg_info);
 RZ_API RZ_OWN RzRopGadgetInfo *rz_core_rop_gadget_info_new(ut64 address);
-RZ_IPI RzRopRegInfo *rz_core_rop_reg_info_dup(RzRopRegInfo *src);
-RZ_IPI void rz_core_rop_reg_info_free(RzRopRegInfo *reg_info);
+RZ_API RzRopRegInfo *rz_core_rop_reg_info_dup(RzRopRegInfo *src);
+RZ_API void rz_core_rop_reg_info_free(RzRopRegInfo *reg_info);
 RZ_IPI RzRopRegInfo *rz_core_rop_reg_info_new(const RzCore *core, const RzILEvent *evt, ut64 init_val, ut64 new_val);
```

Comment:
please convert this to `RZ_API` from `RZ_IPI`. there should be no `RZ_IPI` in the headers inside the `include` folder.

Reply:
be aware that this needs to be done also in the `.c` file.